frontend/apps/reader/modules/readersearch.lua

```diff
@@ -144,6 +145,13 @@ function ReaderSearch:onShowSearchDialog(text, direction)
                 end
                 -- Don't add result pages to location ("Go back") stack
                 neglect_current_location = true
+            else
+                UIManager:show(
+                    Notification:new{
+                        text = T(_("'%1' not found"), text),
+                        timeout = 3,
```

Comment:
Please get rid of the timeout - I made them all be 2 seconds in https://github.com/koreader/koreader/commit/8f79c662cfc5e1d2fb714c7dcef7c3c9318a62ef , for consistency, your cup of tea :)